src/web/components/CMP.tsx

```diff
@@ -24,10 +29,12 @@ export const CMP = () => {
     return (
         <>
             {show && (
-                <ConsentManagementPlatform
-                    source="dcr"
-                    onClose={() => setShow(false)}
-                />
+                <Suspense fallback={<></>}>
```

Comment:
we fallback to a react fragment instead of a div for the tests

Reply:
Why do we need suspense? Just for the tests?

Reply:
I thought this was for tests, but looks we should always use React.lazy with Suspense